cfgov/unprocessed/css/enhancements/forms.less

```diff
@@ -300,6 +300,80 @@ legend {
 }
 
 /* topdoc
+  name: Input with button bar
```

Comment:
I like how you extended this. Is there room in your sprint to also get this into Capital Framework here => https://cfpb.github.io/capital-framework/components/cf-forms/#simple-input-with-a-button